resources/home/dnanexus/utils/io.py

```
@@ -0,0 +1,366 @@
+        json.dump(template_copy, outfile, indent=4)
+
+    session_file = dxpy.upload_local_file(
+        output_name,
+        folder=f"{job_output}/igv_sessions",
+        tags=[expiry_date],
+        wait_on_close=True,
+    )
+
+    session_file_id = session_file.get_id()
+
+    return session_file_id
```

Ruff (0.8.0)

Unnecessary assignment to `session_file_id` before `return` statement

Remove unnecessary assignment

(RET504)